src/main/java/net/mcreator/ui/modgui/IBlocklyPanelHolder.java

```diff
@@ -36,4 +36,16 @@ default BlocklyPanel getSpecificBlocklyPanel(BlocklyEditorType type) {
 		return null;
 	}
 
+	default boolean isInitialXMLValid() {
+		return true;
+	}

```

Comment:
This method should take the type parameter to match the existing part of this interface (not sure about the other one, though, should probably be implemented as `HashMap<BlocklyEditorType, List<BlocklyChangedListener>>` in the editors' classes and take such parameter as well):
```suggestion
	default boolean isInitialXMLValid(BlocklyEditorType type) {
		return true;
	}
```

Reply:
Right now the only use case of this is by tests where all editors need to have initial XML valid.

If needed the system will be extended in the future.